src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,98 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lotto {
+    public static final String IS_NOT_LOTTO_SIZE = "번호가 6개가 아닙니다.";
+    private static final String INPUT_IS_NOT_NUMBER = "입력된 값이 숫자가 아닙니다.";
+    public static final String IS_NOT_LOTTO_NUMBER = "로또번호가 1~45가 아닙니다.";
+    private static final int LOTTO_MIN_NUM = 1;
+    private static final int LOTTO_MAX_NUM = 45;
+    private final List<Integer> lottoNumbers;
+
+    public static Lotto autoLotto() {
+        return new Lotto(makeAutoLotto());
+    }
+
+    public Lotto(String[] inputNumbers) {
+        List<Integer> lotto = splitInput(inputNumbers);
+        validateLottoSetSize(lotto);
+        this.lottoNumbers = makeSortLotto(lotto);
+    }
+
+    public Lotto(List<Integer> lottoNumbers) {
+        validateLottoSetSize(lottoNumbers);
+        this.lottoNumbers = makeSortLotto(lottoNumbers);
+    }
```

Comment:
```suggestion
    public Lotto(String[] inputNumbers) {
        this(splitInput(inputNumbers));
    }

    public Lotto(List<Integer> lottoNumbers) {
        validateLottoSetSize(lottoNumbers);
        this.lottoNumbers = makeSortLotto(lottoNumbers);
    }
```
생성자의 가장 큰 목적은 안정적인 객체 생성인데요. 객체 생성에 검증이 중복된다고 했을 때 누락의 여지가 있으므로 안정적인 객체 생성이 가능한 주생성자를 두고, 이를 활용하는 부생성자를 만들어 안정적인 객체 생성을 보장할 수 있습니다 :)